understanding/22/focus-appearance-minimum.html

```diff
@@ -20,20 +20,20 @@ <h2>Focus Appearance (Minimum) Success Criterion text</h2>
         <blockquote>
             <p class="conformance-level">AA</p>
         
-            <p>When <a>user interface components</a> receive keyboard focus, an area of the focus indicator meets the following:</p>
+            <p>When a <a>user interface component</a> has keyboard focus, the focus indicator meets all the following:</p>
             <ul>
-                <li><strong>Minimum area:</strong> The area is either:
+                <li><strong>Contrasting area:</strong> The focus indicator has an area with a contrast ratio of at least 3:1 between the colors when the component is focused and when it is not focused, and that area is at least as large as either:
                 <ul>
-                    <li>at least as large as the area of a 1 <a>CSS pixel</a> thick <a>perimeter</a> of the unfocused component, or</li>
-                    <li>at least as large as the area of a 4 CSS pixel thick line along the shortest side of a <a>minimum bounding box</a> of the unfocused component, and no part of the area is thinner than 2 CSS pixels.</li>
+                    <li>the area of a 1 <a>CSS pixel</a> thick <a>minimum bounding box</a> around the content of the component, or</li>
```

Comment:
Minor thing – perimeter is still a defined term but no longer appears in the success criteria.